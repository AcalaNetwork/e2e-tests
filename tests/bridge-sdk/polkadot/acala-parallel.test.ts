import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'acala',
  to: 'parallel',
  token: 'PARA',
})

bridgeSDKTest({
  from: 'acala',
  to: 'parallel',
  token: 'ACA',
})
