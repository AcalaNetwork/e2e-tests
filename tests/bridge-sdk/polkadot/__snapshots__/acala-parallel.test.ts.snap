// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'ACA' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 8)",
  "toChain": "(rounded 102)",
}
`;

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'ACA' > fee 1`] = `"(rounded 0.1)"`;

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'ACA' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 100,
}
`;

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'PARA' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 8,
  "toChain": "(rounded 1000)",
}
`;

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'PARA' > fee 1`] = `"(rounded 1)"`;

exports[`'acala' to 'parallel' using bridgeSDK cross-chain 'PARA' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 10,
  "toChain": 1000,
}
`;
